Easydict/App/Localizable.xcstrings

```diff
@@ -629,22 +629,6 @@
         }
       }
     },
-    "clear_input" : {
```

Comment:
Align with SwiftUI Version, so the key is useless now. ![CleanShot 2024-02-02 at 16 48 43@2x](https://github.com/tisfeng/Easydict/assets/1056803/89e1277b-f833-48d1-ab69-06440ec2880c)

Reply:
ok